test/e2e/hive_manager.go

```diff
@@ -0,0 +1,42 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Hive manager", func() {
+	Describe("CreateNamespace", func() {
+		var ns *corev1.Namespace
+		It("Should be created successfully", func() {
+			var err error
+			ns, err = clients.HiveClusterManager.CreateNamespace(context.Background())
```

Comment:
I do not see a benefit of this test: we already have an unit test for `CreateNamespace` which is sufficent for ensuring that this method works as expected.

We should not be testing metohds of the hive manager separately in E2Es: we need to test whether the whole flow of cluster creation/adoption works end to end. From customer's request to successfull cluster creation/adoption via Hive.

Reply:
@m1kola I entirely agree that this test is almost useless in terms of coverage of Hive-related processes.
But I consider this as something that proves that our e2e infra is ready for Hive testing to some extent.